cpp/src/io/parquet/page_data.cu

```diff
@@ -2027,11 +2035,14 @@ __global__ void __launch_bounds__(decode_block_size) gpuDecodePageData(
     return;
   }
 
+  auto const data_type = s->col.data_type & 7;
```

Comment:
`data_type` is not used that often in other functions. Let me have a further look.

Reply:
Can we use `parquet::Type` directly instead of `uint16_t`?
https://github.com/rapidsai/cudf/blob/41f0caf53662cfde8146647574e705982eb558b1/cpp/src/io/parquet/parquet_gpu.hpp#L266-L267

Reply:
(for some reason) data_type holds the type in the last three bits and length in the rest of the number. Using Type here would make this data member even more misleading.